### PR TITLE
Add MergeSpecsIgnorePathConflict

### PR DESCRIPTION
**Pull request by mbohlool**
MergeSpecsIgnorePathConflict is a new merge method that ignores path conflicts. It is useful if we want the destination has priority over source in serving paths.

### CODE REVIEW

File: pkg/aggregator/aggregator.go [hunk 1]
```diff
@@ -209,19 +209,43 @@ func renameDefinition(s *spec.Swagger, old, new string) {
 	delete(s.Definitions, old)
 }
 
-// MergeSpecsOrFailOnConflict is differ from MergeSpecs as it fails if there is
+// MergeSpecsIgnorePathConflict is the same as MergeSpecs except it will ignore any path conflicts.
```
Comment (sttts) [inline pkg/aggregator/aggregator.go]: ignoring by what? by keeping the entry for dest or source?

File: pkg/aggregator/aggregator.go [hunk 2]
```diff
@@ -209,19 +209,43 @@ func renameDefinition(s *spec.Swagger, old, new string) {
...
 
-func mergeSpecs(dest, source *spec.Swagger, renameModelConflicts bool) (err error) {
+func mergeSpecs(dest, source *spec.Swagger, renameModelConflicts, ignorePathConflicts bool) (err error) {
+
+	specCloned := false
+	if ignorePathConflicts {
+		keepPaths := []string{}
+		for k := range source.Paths.Paths {
+			if _, found := dest.Paths.Paths[k]; !found {
+				keepPaths = append(keepPaths, k)
```
Comment (sttts) [inline pkg/aggregator/aggregator.go]: so dest wins, right? Should be in the comment above.

**Comment by sttts** [comment]
Just a comment about godoc. After fixing that, lgtm.

File: pkg/aggregator/aggregator.go (hunk 1 above)
Comment (mbohlool) [reply pkg/aggregator/aggregator.go]: fixed.

**Pull request merged**